specification/communication/Communication.JobRouter/routes.tsp

```diff
@@ -0,0 +1,366 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.Core.Traits;
+
+namespace AzureCommunicationRoutingService;
+
+alias ServiceTraits = NoRepeatableRequests &
+  NoConditionalRequests & NoClientRequestId;
+alias PatchOperationTraits = NoRepeatableRequests & NoClientRequestId & SupportsRouterConditionalPatchUpdateRequests & RouterReturnConditionalResponseHeaders;
+alias GetOperationTraits = ServiceTraits & RouterReturnConditionalResponseHeaders;
+
+alias Operations = Azure.Core.ResourceOperations<ServiceTraits>;
+alias GetOperations = Azure.Core.ResourceOperations<GetOperationTraits>;
+alias PatchOperations = Azure.Core.ResourceOperations<PatchOperationTraits>;
+
+interface JobRouterAdministrationOperations {
+  #suppress "@azure-tools/typespec-azure-core/no-explicit-routes-resource-ops" "Codegen cannot import prefixes from namespace"
```

Comment:
Can you explain what support is missing here? Is there an open issue for this?

Reply:
Yes I raised it with the respective folks. [Teams post (internal)](https://teams.microsoft.com/l/message/19:906c1efbbec54dc8949ac736633e6bdf@thread.skype/1698882012944?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1698882012944&teamName=Azure%20SDK&channelName=TypeSpec%20Discussion%20%F0%9F%90%AE&createdTime=1698882012944) for more context

Reply:
Recent discussion there indicates that there is a solution for the REST API and the SDK gen fix is in the works, so I think we should drop the explicit route and suppression here.

Reply:
Have removed @routing decorator and related suppressions from all methods with the exception of `getInQueuePosition` and `getQueueStatistics` which defines [routes for singletons](https://github.com/Azure/azure-rest-api-specs/issues/25605#issuecomment-1736265997)